client/src/components/Artists/GoogleMap.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import { Map, Marker, GoogleApiWrapper } from 'google-maps-react'
+class GoogleMap extends Component {
+ render() {
+   return (
+      <>
+         <Map
+            google={this.props.google}
+            zoom={19}
+            style={{
+               width: '350px',
+               height: '300px'
+            }}
+            initialCenter={{
+               lat: 30.2656,
+               lng: -97.7497
+         }}>
+            <Marker />
+         </Map>
+      </>
+   );
+ }
+};
+
+export default GoogleApiWrapper({
+  apiKey: ('AIzaSyCr6DX1zFp38DKxr9C-5grSmt1IVDGfCAo')
```

Comment:
Remove api key and store your secrets in an another file. I would recommend re-generating this as well since it could be compromised.